### PR TITLE
autojump: add page

### PR DESCRIPTION
**Pull request by chuanconggao**

### CODE REVIEW

File: pages/common/autojump.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# autojump
+
+> Automatically jump to the directory you mostly visited.
```
Comment (waldyrious) [inline pages/common/autojump.md]: Perhaps rephrase to "the directory you visited the most", or "the most visited directory".

File: pages/common/autojump.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Jump to a directory that contains the pattern.
```
Comment (waldyrious) [inline pages/common/autojump.md]: Suggestion: "...the given pattern."

File: pages/common/autojump.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+`j {{pattern}}`
+
+- Jump to a sub-directory of current directory that contains the pattern.
```
Comment (waldyrious) [inline pages/common/autojump.md]: Same as above.

File: pages/common/autojump.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+`jc {{pattern}}`
+
+- Remove non-exist directories from the database.
```
Comment (waldyrious) [inline pages/common/autojump.md]: non-existing

File: pages/common/autojump.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+`j --purge`
+
+- Show database entries.
```
Comment (waldyrious) [inline pages/common/autojump.md]: Same as above, e.g. "Show the entries in the autojump database".

**Comment by felixonmars** [comment]
:+1: Looks good now.

**Comment by felixonmars** [comment]
There is an issue regarding the naming though. The software is called `autojump`, but the actual commands are j and jc. I would suggest to name the page according to the commands, since we did that for most other pages (consider a single huge `coreutils` page vs separate pages for commands in coreutils including `ls`, `mv`, `df`, ...).

**Comment by chuanconggao** [comment]
@felixonmars Actually `autojump` can also be used directly. `j` and `jc` are just two aliases.

**Comment by waldyrious** [comment]
@chuanconggao maybe then that can be mentioned in the main description :)

**Comment by chuanconggao** [comment]
Just updated the description.

**Comment by igorshubovych** [comment]
Thank you!

**Pull request merged**